hyperspy/component.py

```diff
@@ -1027,8 +1020,6 @@ def _component2plot(self, axes_manager, out_of_range2nans=True):
         s = self.__call__()
         if not self.active:
             s.fill(np.nan)
-        if self.model.signal.metadata.Signal.binned is True:
-            s *= self.model.signal.axes_manager.signal_axes[0].scale
```

Comment:
This seems like an important change? Was this relevant just for the convolution plot or plots in general?

Reply:
What do you mean? Regarding binned data, there is no change in this PR.

Reply:
Then why does the PR remove lines 1030 and 1031?

Reply:
Plotting component has been reworked and these lines needed to be removed to be correct. If you read the code, you will understand.

Reply:
Gotcha! Was just making sure you hadn't accidentally removed something :)